cmake/multiap-helpers.cmake

```diff
@@ -29,12 +29,12 @@ if (${FORCE_COLORED_OUTPUT})
     endif ()
 endif ()
 
-if (NOT ${PROJECT}_REVISION)
+if (NOT REVISION)
```

Comment:
PRPLMESH_REVISION is more appropriate IMO (nitpick)

Reply:
Yes I considered that. But then I didn't think the prefix is very useful - we're in the prplmesh project, so obviously it's about prplmesh... Why don't we have PRPLMESH_INSTALL_PATH then for example?